time.py

```diff
@@ -0,0 +1,12 @@
+input1=int(input("enter the total no of seconds :"))
```

Comment:
totalSeconds=int(input("Enter total seconds for conversion: "))
if(totalSeconds<0):
     print("Invalid seconds")